### PR TITLE
feat(widget-configurator): google analytics events

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Added `@cowprotocol/analytics` to the widget configurator.
Currently we track three events:
1. When user connect to a wallet `{category: 'Widget configurator', event: 'Connect Wallet'}`
2. When user opens Embed code modal `{category: 'Widget configurator', event: 'View Code'}`
3. When user copy embed code `{category: 'Widget configurator', event: 'Copy Code'}`

  # To Test

1. Open PR preview
2. Do actions from the list above
3. Check that events went out to GA
4. GA should have the same ID as at CoWSwap

### CODE REVIEW

**Comment by shoom3301** [comment]
Connect wallet event:
image

View embed code:
image

Copy embed code:
image

File: apps/widget-configurator/src/app/analytics.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  sendEvent({
+    category: Category.WIDGET_CONFIGURATOR,
+    action: 'Connect wallet',
+  })
+}
+
+export function viewEmbedCode() {
+  sendEvent({
+    category: Category.WIDGET_CONFIGURATOR,
+    action: 'View embed code',
```
Comment (anxolin) [inline apps/widget-configurator/src/app/analytics.ts:13]: should we remove the embed word.

Also, should be embedded? right? Anyways, IMO "View code" is enough

File: apps/widget-configurator/src/app/analytics.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  })
+}
+
+export function copyEmbedCode() {
+  sendEvent({
+    category: Category.WIDGET_CONFIGURATOR,
+    action: 'Copy embed code',
```
Comment (anxolin) [inline apps/widget-configurator/src/app/analytics.ts:20]: same here

File: apps/widget-configurator/vite.config.ts [hunk 3]
```diff
@@ -14,24 +17,31 @@ const plugins = [
     },
     protocolImports: true,
   }),
+  macrosPlugin(),
```
Comment (anxolin) [inline apps/widget-configurator/vite.config.ts:20]: why did you need macros in this PR?

File: apps/widget-configurator/src/app/analytics.ts (hunk 1 above)
Comment (shoom3301) [reply apps/widget-configurator/src/app/analytics.ts]: Fixed

File: apps/widget-configurator/src/app/analytics.ts (hunk 2 above)
Comment (shoom3301) [reply apps/widget-configurator/src/app/analytics.ts]: Fixed

File: apps/widget-configurator/vite.config.ts (hunk 3 above)
Comment (shoom3301) [reply apps/widget-configurator/vite.config.ts:20]: image

Without the plugin we get the error

Comment (shoom3301) [reply apps/widget-configurator/vite.config.ts:20]: I see it came from `ms.macro`
